### DISCUSSION TITLE
error running tutorial - (0x800700AA) The requested resource is in use

### DISCUSSION THREAD

**Discussion by reichunter1656** [microsoft/perfview#1772] [Q&A]
Following the documentation to run the tutorial but running the command almost immediately fails with this log. I am running from my personal laptop (windows 11). I have attached a screenshot of 'Run a command' window where I press 'Run'.

![2022-12-30_15h26_38](https://user-images.githubusercontent.com/5817870/210109502-fe58e931-65c6-4a64-9fe4-dbe0228a7707.png)

Has anyone came across this error, the only thing is I have only one drive on my laptop (C:), but I did try to run the tutorial in admin mode with the same error. Any help is appreciated. Here is the log:

> Started with command line: "C:\PerfView\New folder\PerfView.exe" 
> PerfView Version: 3.0.6  BuildDate: Mon, 07 Nov 2022 20:31:05 GMT
> PerfView Start Time 12/30/2022 3:17:28 PM
> Started: View
> Warning: PdbScope not found at C:\PerfView\New folder\PerfViewExtensions\PdbScope.exe
> Disabling the Image Size Menu Item.
> Warning: ILSize not found at C:\PerfView\New folder\PerfViewExtensions\ILSize.dll
> Disabling the IL Size Menu Item.
> Completed: View   (Elapsed Time: 0.039 sec)
> Looking up topic Tutorial in Users Guide.
> Collection Dialog open.
> Started: Running: Tutorial.exe...  See log for output.
> [Kernel Log: C:\PerfView\New folder\PerfViewData.kernel.etl]
> Kernel keywords enabled: Default
> Aborting tracing for sessions 'NT Kernel Logger' and 'PerfViewSession'.
> Ensuring .NET Allocation profiler not installed.
> Detected 64 bit system, removing 32 bit keys.
> Completed: Running: Tutorial.exe...  See log for output.   (Elapsed Time: 0.371 sec)
> Exception Occurred: System.Runtime.InteropServices.COMException (0x800700AA): The requested resource is in use. (Exception from HRESULT: 0x800700AA)
>    at System.Runtime.InteropServices.Marshal.ThrowExceptionForHRInternal(Int32 errorCode, IntPtr errorInfo)
>    at Microsoft.Diagnostics.Tracing.Session.TraceEventSession.EnableKernelProvider(Keywords flags, Keywords stackCapture)
>    at PerfView.CommandProcessor.Start(CommandLineArgs parsedArgs) in D:\a\_work\1\s\src\PerfView\CommandProcessor.cs:line 500
>    at PerfView.CommandProcessor.Run(CommandLineArgs parsedArgs) in D:\a\_work\1\s\src\PerfView\CommandProcessor.cs:line 113
>    at PerfView.MainWindow.<>c__DisplayClass23_0.<ExecuteCommand>b__0() in D:\a\_work\1\s\src\PerfView\MainWindow.xaml.cs:line 1350
>    at PerfView.StatusBar.<>c__DisplayClass22_0.<StartWork>b__0() in D:\a\_work\1\s\src\PerfView\GuiUtilities\StatusBar\StatusBar.xaml.cs:line 254
> An exceptional condition occurred, see log for details.

**Comment by Bug-001** [comment]
Are you using Intel Core processors? If yes, it has something to do with Windows Defender. Windows Defender will randomly start hardware performance counters (HPCs). So the requested HPC resource is in use. For more information about how Windows Defender interacts with HPCs, go to this link: [https://www.techpowerup.com/295877/windows-defender-can-significantly-impact-intel-cpu-performance-we-have-the-fix?cp=4.](https://www.techpowerup.com/295877/windows-defender-can-significantly-impact-intel-cpu-performance-we-have-the-fix?cp=4.)

And issue #1723 will give you more information.

To solve this problem, try the Counter Control tool mentioned in the above link, or simply turn off Windows Defender's real-time protection.
